app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+# before_filter :configure_sign_up_params, only: [:create]
+# before_filter :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  # def new
+  #   super
+  # end
+
+  # POST /resource
+  # def create
+  #   super
+  # end
+
+  GET /resource/edit
```

Comment:
ambiguous first argument; put parentheses or a space even after the operator<br>unknown regexp options: dt